CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Added
+- Support for custom options of type input and area
```

Comment:
One more thing, this is not really correct. The support was already there, and we actually most likely support all options, we just need to add tests for them.